pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "pixelbar-pyLedController"
+version = "0.1.0"
+description = ""
+authors = ["aldo@fieldofview.com"]
+
+[tool.poetry.dependencies]
+python = "^3.10"
```

Comment:
The pi we're running on actually has python 3.9.2, but I developed this on 3.10. Is that legal in poetry?

Reply:
Good point! W can update this in poetry, and open a new PR